tests/boardfarm_plugins/run_bf.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+
+scriptdir=$(dirname "$(readlink -f "${0}")")
+bf_plugins_dir=${scriptdir}
+bf_dir=$(realpath "${scriptdir}/../../../boardfarm/boardfarm")
+prplMesh_dir=$(realpath "${scriptdir}/..")
+
+if [ -n "${PYTHONPATH}" ]; then
+   PYTHONPATH="${bf_dir}:${bf_plugins_dir}:${prplMesh_dir}:${PYTHONPATH}"
+else
+   PYTHONPATH="${bf_dir}:${bf_plugins_dir}:${prplMesh_dir}"
+fi
+export PYTHONPATH
+
+exec bft -c "${bf_plugins_dir}"/boardfarm_sample_plugin/prplmesh_config.json -n prplmesh_dummy -x sample-suite
```

Comment:
Does the boardfarm setup really have to be run? Can't we skip it by using
```suggestion
exec ${bf_dir}/bft -c "${bf_plugins_dir}"/boardfarm_sample_plugin/prplmesh_config.json -n prplmesh_dummy -x sample-suite
```

Reply:
> Does the boardfarm setup really have to be run? Can't we skip it by using

We need install boardfarm to run it.
I've spent some time to investigate how we can bypass that limitation - it requires changes of boardfarm sources, which may be very time consuming.

Reply:
No we don't.

The only thing that is needed is to install the requirements. So `pip3 install -r requirements.txt` is sufficient. Or alternatively, what I did, install the required packages using the distro package manager.